### PR TITLE
Info about MFA

### PR DESCRIPTION
**Pull request by matthewshaver**
## What are you changing in this pull request and why?

See: https://dbt-labs.slack.com/archives/C05CJT5MJF2/p1729019209832589

## Checklist
- [ ] I have reviewed the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [ ] The topic I'm writing about is for specific dbt version(s) and I have versioned it according to the [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and/or [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content) guidelines.
- [ ] I have added checklist item(s) to this list for anything anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."
- [ ] The content in this PR requires a dbt release note, so I added one to the [release notes page](https://docs.getdbt.com/docs/dbt-versions/dbt-cloud-release-notes).

---
🚀 Deployment available! Here are the direct links to the updated files:

- https://docs-getdbt-com-git-mfa-dbt-labs.vercel.app/docs/cloud/manage-access/mfa
- https://docs-getdbt-com-git-mfa-dbt-labs.vercel.app/docs/dbt-versions/release-notes

### CODE REVIEW

File: website/docs/docs/dbt-versions/release-notes.md [hunk 1]
```diff
@@ -20,6 +20,7 @@ Release notes are grouped by month for both multi-tenant and virtual private clo
 
 ## October 2024
  
+- **Behavior change:** [Multi-factor authentication](/docs/cloud/manage-acces/mfa) is now enforced on all users who log in with username and password credentials.
```
Comment (matthewshaver) [inline website/docs/docs/dbt-versions/release-notes.md]:
```suggestion
- **Behavior change:** [Multi-factor authentication](/docs/cloud/manage-access/mfa) is now enforced on all users who log in with username and password credentials. 
```

**Review by nehahystad — COMMENTED** [review]
Would change this "Users who have not MFA enabled" to "Users who have not set up MFA" in the call out. Otherwise LGTM!

File: website/docs/docs/cloud/manage-access/mfa.md [hunk 2]
```diff
@@ -7,6 +7,13 @@ sidebar: null
 
 # Multi-factor authentication
 
+:::Important
+
+
+dbt Cloud enforces multi-factor authentication (MFA) for all users with username and password credentials. Users who have not MFA enabled will see a notification asking them to configure one of the supported methods when they log in. If they do not, they will have to configure MFA on the next login or be unable to access dbt Cloud.
```
Comment (matthewshaver) [inline website/docs/docs/cloud/manage-access/mfa.md]:
```suggestion
dbt Cloud enforces multi-factor authentication (MFA) for all users with username and password credentials. Users who have not set up MFA will see a notification asking them to configure one of the supported methods when they log in. If they do not, they will have to configure MFA on the next login or be unable to access dbt Cloud.
```

File: website/docs/docs/cloud/manage-access/mfa.md [hunk 3]
```diff
@@ -7,6 +7,13 @@ sidebar: null
 
 # Multi-factor authentication
 
+:::Important
```
Comment (matthewshaver) [inline website/docs/docs/cloud/manage-access/mfa.md]:
```suggestion
:::important
```

File: website/docs/docs/cloud/manage-access/mfa.md [hunk 4]
```diff
@@ -7,6 +7,13 @@ sidebar: null
 
 # Multi-factor authentication
 
+:::important
+
+
+dbt Cloud enforces multi-factor authentication (MFA) for all users with username and password credentials. Users who have not set up MFA will see a notification asking them to configure one of the supported methods when they log in. If they do not, they will have to configure MFA on the next login or be unable to access dbt Cloud.
```
Comment (runleonarun) [inline website/docs/docs/cloud/manage-access/mfa.md]: Will they need to write a support ticket to get unblocked? Suggest using second person for clarity. 
```suggestion
dbt Cloud enforces multi-factor authentication (MFA) when anyone signs in with username and password credentials. If MFA is not set up, you'll be asked to configure one of the supported methods when you log in. If you don't configure MFA after your second login and notification, you'll be unable to access dbt Cloud.
```

File: website/docs/docs/dbt-versions/release-notes.md [hunk 5]
```diff
@@ -19,7 +19,8 @@ Release notes are grouped by month for both multi-tenant and virtual private clo
 \* The official release date for this new format of release notes is May 15th, 2024. Historical release notes for prior dates may not reflect all available features released earlier this year or their tenancy availability.
 
 ## October 2024
-
+ 
+- **Behavior change:** [Multi-factor authentication](/docs/cloud/manage-access/mfa) is now enforced on all users who log in with username and password credentials.
```
Comment (runleonarun) [inline website/docs/docs/dbt-versions/release-notes.md:23]:
```suggestion
- **Behavior change:** [Multi-factor authentication](/docs/cloud/manage-access/mfa) is now enforced when anyone signs in with username and password credentials. 
```

**Review by runleonarun — APPROVED** [review]
Approved with some wording suggestions!

File: website/docs/docs/cloud/manage-access/mfa.md (hunk 4 above)
Comment (matthewshaver) [reply website/docs/docs/cloud/manage-access/mfa.md]: There is no unblocked. They have to configure MFA at that point

Comment (matthewshaver) [inline website/docs/docs/cloud/manage-access/mfa.md]:
```suggestion
dbt Cloud enforces multi-factor authentication (MFA) for all users with username and password credentials. If MFA is not set up, you will see a notification bar prompting you to configure one of the supported methods when you log in. If you do not, you will have to configure MFA upon subsequent logins, or you will be unable to access dbt Cloud.
```

**Pull request merged**